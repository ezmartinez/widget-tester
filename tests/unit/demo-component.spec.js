const React = require('react');
const { shallow } = require('enzyme');
const DemoComponent = require('../../app/components/DemoComponent');

test('DemoComponent should render correctly', () => {
  const wrapper = shallow(<DemoComponent />);
  expect(wrapper).toMatchSnapshot();
});
