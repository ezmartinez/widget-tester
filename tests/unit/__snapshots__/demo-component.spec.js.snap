// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DemoComponent should render correctly 1`] = `
<span>
  This is a Demo component
</span>
`;
